src/components/molecules/PlanRoom/RoomTables/Table/index.tsx

```diff
@@ -90,6 +97,51 @@ const TableList = () => {
     [MoveTable],
   );
 
+  ClearBoard = () => {
+    while (numberTable > -1) {
```

Comment:
It should rather be called `tableNumber`.
![image](https://user-images.githubusercontent.com/48626088/123448920-7c021f00-d5db-11eb-8caf-81bb6b81154d.png)

Reply:
right